src/settings.rs

```diff
@@ -152,15 +152,26 @@ impl Settings {
             .expect("unable to set capatcha.enable_stats default config");
 
         if let Ok(path) = env::var("MCAPTCHA_CONFIG") {
-            let absolute_path =
-                Path::new(&path).canonicalize().unwrap().to_str().unwrap();
-            log::info!("{}", format!("Loading config file from {}", absolute_path));
-            s.merge(File::with_name(absolute_path))?;
+            let absolute_path = Path::new(&path).canonicalize().unwrap();
+            log::info!(
```

Comment:
`format!` is redundant here, `log::info!` macro is capable of formatting, just like `format!` and `print!`